test/specs/configs/proxy.yaml

```diff
@@ -1,4 +1,4 @@
 version: 1
 strains:
-  default:
-    origin: https://www.example.com
\ No newline at end of file
+  - name: default
+    origin: https://www.example.com
```

Comment:
@trieloff this is not valid per your schemas:

```
 12) Validator Tests
       proxy.yaml is valid:
     Error: Invalid configuration:
Invalid Strain .strains should be object: type([{"name":"default","origin":"https://www.example.com","directoryIndex":"index.html"}], {"type":"object"})
Proxy Strain .strains[0] has unknown property 'directoryIndex'
Runtime Strain .strains[0] has unknown property 'origin'
Runtime Strain .strains[0] should have required property 'code'
Runtime Strain .strains[0] should have required property 'content'
Runtime Strain .strains[0] should have required property 'static'
Invalid Strain .strains[0] must be either a Runtime Strain or a Proxy Strain
Invalid Strain .strains should contain a valid item: contains([{"name":"default","origin":"https://www.example.com","directoryIndex":"index.html"}], {})
Strains .strains must be either a Runtime Strain or a Proxy Strain

data.strains should be object, data.strains[0] should NOT have additional properties, data.strains[0] should NOT have additional properties, data.strains[0] should have required property 'code', data.strains[0] should have required property 'content', data.strains[0] should have required property 'static', data.strains[0] should match exactly one schema in oneOf, data.strains should contain a valid item, data.strains should match exactly one schema in oneOf
      at ConfigValidator.assetValid (src/ConfigValidator.js:15:494)
      at validate (test/validator.test.js:26:25)
      at <anonymous>
```

can you have a look?

Reply:
Yes, I'll take a look.